internal/operators/cnv/manifest.go

```diff
@@ -14,6 +15,8 @@ const (
 
 	upstreamSourceName   string = "community-kubevirt-hyperconverged"
 	downstreamSourceName string = "kubevirt-hyperconverged"
+
+	discoverableDiskSizeThresholdGi int64 = 50
```

Comment:
suggesting to make it configurable (env var) it will be easier to change, instead of releasing a new version.

Reply:
Are env variables exposed to end users who install a cluster?

Reply:
No, it is a configuration that can be changed from the deployment yaml.
This is a validation, meaning that if the user have an additional smaller disk, he will not be able to continue with the installation.

Reply:
So frankly, I do not see a point in making the size configurable. We may realize in the future that there are users who do not need storage for SNO+CNV; but then we should not lower discoverableDiskSizeThresholdGi, we'd need to drop the entire validation.

Reply:
The price in the code is not that high, comparing to have to release a new image.
Also it allows QE to tweak values if needed.

Need something like that:
`OCSMinDiskSizeGB                int64             envconfig:"OCS_MIN_DISK_SIZE_GB" default:"25"`

Reply:
Made is a configurable env var under the CNV Config

Reply:
IMHO the most (only?) interesting config is to skip HPP validation and configuration if the value is 0.

Reply:
Added opt-out-of-installing HPP env var